docs/doc_sources/api_reference/dpctl/sycl_usm_array_interface.rst

```diff
@@ -32,23 +32,23 @@ Python dictionary with the following fields:
     is read-only).
 
 ``"strides"``
-    an optional tuple of integers describing number of array elements needed to jump
+    An optional tuple of integers describing number of array elements needed to jump
     to the next array element in the corresponding dimensions. The default value of ``None``
     implies a C-style contiguous (row-major compact) layout of the array.
 
 ``"offset"``
-    an optional Python integer encoding offset in number of elements from the pointer
+    An optional Python integer encoding offset in number of elements from the pointer
     provided in ``"data"`` field to the array element with zero indices. Default: `0`.
 
 ``"syclobj"``
     Python object from which SYCL context to which represented USM allocation is bound.
 
     ==============================================  =======================================
-    Filter selector string                          Platform's default context for platform
+    Filter selector string                          Platform's default context for the platform
```

Comment:
This does not read correctly:

> Platform's default context for the platform the SYCL device selected by the
:ref:`filter selector string <filter_selector_string>` it is a part of.
 
How about we reword it:

The default context of the platform selected by :ref:`filter selector string <filter_selector_string>`.

Reply:
@oleksandr-pavlyk  I updated with your revised text. https://github.com/IntelPython/dpctl/pull/1659/commits/0fa480be593b5c40279d8608317339a3a16f4670
Please resolve this conversation if it looks good to you.